fixtures/u2_func_specific_kw_optional_param.rb.sem.uast

```diff
@@ -0,0 +1,75 @@
+{ '@type': "module",
+   '@role': [Identifier, Module, Statement],
+   file: { '@type': "uast:FunctionGroup",
+      '@pos': { '@type': "uast:Positions",
+         start: { '@type': "uast:Position",
+            offset: 0,
+            line: 1,
+            col: 1,
+         },
+         end: { '@type': "uast:Position",
+            offset: 2,
+            line: 1,
+            col: 3,
+         },
+      },
+      Nodes: [
+         { '@type': "uast:Alias",
+            Name: { '@type': "uast:Identifier",
+               Name: "testfnc1",
+            },
+            Node: { '@type': "uast:Function",
+               Body: { '@type': "uast:Block",
+                  Statements: [
+                     { '@type': "NilNode",
+                        '@role': ['Null'],
+                     },
+                  ],
+               },
+               Type: { '@type': "uast:FunctionType",
+                  Arguments: [
+                     { '@type': "uast:Argument",
+                        '@pos': { '@type': "uast:Positions",
+                           start: { '@type': "uast:Position",
+                              offset: 13,
+                              line: 1,
+                              col: 14,
+                           },
+                           end: { '@type': "uast:Position",
+                              offset: 13,
+                              line: 1,
+                              col: 14,
+                           },
+                        },
+                        Init: { '@type': "int",
+                           '@token': 1,
+                           '@role': [Expression, Literal, Number, Primitive],
+                           '@pos': { '@type': "uast:Positions",
+                              start: { '@type': "uast:Position",
+                                 offset: 16,
+                                 line: 1,
+                                 col: 17,
+                              },
+                              end: { '@type': "uast:Position",
+                                 offset: 16,
+                                 line: 1,
+                                 col: 17,
+                              },
+                           },
+                        },
+                        MapVariadic: false,
```

Comment:
This should probably be set to `true`, since it's a kwarg

Reply:
No, this is an argument with a name and a default value, `MapVariadic` is for formal arguments defining auto constructed maps from the actual named arguments on the call, just like varargs is for formal arguments defining lists,  Ruby doesn't have a read translation for this but in python "kwargs" here for have the MapVariadic

```python
def mapVar(**kwargs):
  print(kwargs["a"])
  print(kwargs["b"])
  print(kwargs["c"])

mapVar(a =1, b=2, c=3)
```